values.yaml

```
@@ -169,6 +177,10 @@ resources: {}
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
 nodeSelector: {}
 
+## Service Account Name
+## ref: https://kubernetes.io/docs/concepts/security/service-accounts/
+serviceAccountName: ""
```

should we add the ability to create? A common setup seems to be:
```suggestion
serviceAccount:
  # Specifies whether a service account should be created
  create: true
  # Annotations to add to the service account
  annotations: {}
  # The name of the service account to use.
  # If not set and create is true, a name is generated using the fullname template
  name: ""
```
now we can create the account if needed and insert into the chart.